pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -965,37 +969,43 @@ self: super: {
   hledger = overrideCabal super.hledger (drv: {
     postInstall = ''
       for i in $(seq 1 9); do
-        for j in $data/share/${self.ghc.name}/${pkgs.stdenv.system}-${self.ghc.name}/*/*.$i $data/share/${self.ghc.name}/${pkgs.stdenv.system}-${self.ghc.name}/*/.otherdocs/*.$i; do
+        for j in $data/share/${self.ghc.name}/${ghc_system}-${self.ghc.name}/*/*.$i $data/share/${self.ghc.name}/${ghc_system}-${self.ghc.name}/*/.otherdocs/*.$i; do
```

Comment:
I suppose it's easier to replace the system identifier bit by pattern matching `*`. We shouldn't bother trying to guess what GHC thinks the current system is called.

Reply:
Maybe we shouldn't even try to match the name literally? Is `*-${self.ghc...` good enough? It's not like there are going to be several systems in play at once.

Reply:
Oh, that's what you said, sorry. :-)

I'll do that then.